.github/workflows/release_nightly.yml

```diff
@@ -268,7 +268,7 @@ jobs:
         shell: bash -l {0}
         run: |
           npm run bootstrap
-          npm run build
+          npm run build:full-release
```

Comment:
Quite petty, but maybe rename this to `build:full`?

Reply:
Hmm, I'm not sure about this one... Feels less descriptive to me, while I'm not seeing how it's any better. What do you think, @adrian17?

Reply:
Personally, I lean towards build:release. I understand and somewhat agree with shortening this, but build:release seems to match better with build:debug. As in "this is the debug version. this is the release version". Maybe wait for an opinion from Adrian too though. Calling this the full version, as you mention, feels less descriptive.

Reply:
FWIW:
![image](https://user-images.githubusercontent.com/288816/149030369-c19ddc0b-291e-4f6f-ae37-896af12a6504.png)

So, I am fine with `build:release` as well.

Reply:
> but build:release seems to match better with build:debug
> As in "this is the debug version. this is the release version".

This is my reasoning:
The issue is that these days, "release" means "optimized", not "this is what we use to release the software".

For comparison, these are (very general) build configurations for Firefox:
- max debug, unoptimized; you basically never want to actually run this
- normal optimized build (what `cargo` or Visual Studio would call "release")
- increasingly fancy configurations with LTO, PGO, branding etc - for actual software release or more thorough tests and benchmarks

And these more or less correspond to our configurations:
- max debug, unoptimized; you basically never want to actually run this
- optimized (current `npm run build`)
- optimized with extra debug info/avm tracing (current `npm run build:debug`)
- increasingly fancy configurations with special wasm features

Now, when a dev comes and wants to build it, they most likely want "normal optimized build". But for a Rust programmer, this would probably be `cargo run --release`. But if they apply this logic here and run `build:release`, they get "final-release" and not "development-release" :) Which is why I'd prefer to reserve some more explicit wording for "this is max optimized, not for everyday development".

Reply:
Then pulling from a word you used, what about build:optimized?

Reply:
That isn't perfect either: Even for the plain `build` command now, `-O` is passed to `rustc` and `wasm-opt` is also called on the output either way... :|

Reply:
> Which is why I'd prefer to reserve some more explicit wording for "this is max optimized, not for everyday development".

Although maybe most testing would be done on desktop anyway? Unless it's for a web-specific backend, sure...

Reply:
> That isn't perfect either: Even for the plain build command now, -O is passed to rustc and wasm-opt is also called on the output either way... :|

Yeah, I realized that and deleted my comment suggesting build:optimized. Maybe build:fully-featured, but if the purpose of renaming is to shorten the command, that sort of defeats the purpose.

Reply:
I propose yet another variant: `build:dual-wasm` :| - This one doesn't have `release` it.
And `build:dual-wasm-release` is way too long again?

Reply:
@relrelb Could you please elaborate on what you had against `build:full-release` originally?

Anyway, what do you and @danielhjacobs think about `build:dual-wasm`?

Reply:
I'm fine with dual-wasm

Reply:
I don't have a strong preference to any name (this is a classical example for bikeshedding), I just thought `build:full` is a little more concise and descriptive name. I'm fine with staying on `build:full-release`, changing to `build:dual-wasm`, or even changing to something else.